cli/configure.js

```diff
@@ -1,78 +1,123 @@
-        const fileContents = JSON.stringify(results, null, 2);
-        fs.writeFileSync(fileLocation, fileContents, 'utf-8');
-    })
-    .then(() => {
-        return inquirer.prompt(siteQuestions);
-    })
-    .then(results => {
-        let siteResponses;
-        siteResponses = {
-            analytics: {
-                googleId: results.googleId,
-            },
-            assetDefaults: {
-                title      : results.defaultAssetTitle,
-                description: results.defaultAssetDescription,
-                thumbnail  : results.defaultAssetThumbnail,
-            },
-            auth: {
-                sessionKey: results.sessionKey,
-            },
-            details: {
-                port       : results.port,
-                title      : results.title,
-                host       : results.host,
-                description: results.description,
-                twitter    : results.twitter,
-            },
-            publishing: {
-                primaryClaimAddress     : results.primaryClaimAddress,
-                uploadDirectory         : results.uploadDirectory,
-                thumbnailChannel        : results.thumbnailChannel,
-                thumbnailChannelId      : results.thumbnailChannelId,
-                additionalClaimAddresses: [],
-                disabled                : false,
-                disabledMessage         : 'Please check back soon',
-            },
-        };
-        const fileLocation = Path.resolve(__dirname, '../config/siteConfig.json');
-        const fileContents = JSON.stringify(siteResponses, null, 2);
-        fs.writeFileSync(fileLocation, fileContents, 'utf-8');
-    })
-    .then(() => {
-        return inquirer.prompt(slackQuestions);
-    })
-    .then(results => {
-        const fileLocation = Path.resolve(__dirname, '../config/slackConfig.json');
-        const fileContents = JSON.stringify(results, null, 2);
-        fs.writeFileSync(fileLocation, fileContents, 'utf-8');
-    })
-    .then(() => {
-        console.log('All done!');
-        process.exit(0);
-    })
-    .catch(error => {
-        console.log(error);
-        process.exit(1);
-    });
+  .prompt(loggerQuestions)
+  .then(results => {
+      const fileLocation = Path.resolve(__dirname, '../config/loggerConfig.json');
+      const fileContents = JSON.stringify(results, null, 2);
+      fs.writeFileSync(fileLocation, fileContents, 'utf-8');
+  })
+  .then(() => {
+      return inquirer.prompt(mysqlQuestions);
+  })
+  .then(results => {
+      const fileLocation = Path.resolve(__dirname, '../config/mysqlConfig.json');
+      const fileContents = JSON.stringify(results, null, 2);
+      fs.writeFileSync(fileLocation, fileContents, 'utf-8');
+  })
+  .then(() => {
+    // check for lbrynet connection & retrieve a default address
+    console.log('\nRetrieving primary claim address...');
+    return axios
+      .post('http://localhost:5279', {
```

Comment:
Does the user have to manualy start the daemon for this to work?

Reply:
yeah, the daemon has to be running for it to work otherwise it will throw an error.  It would be great if this CLI would start the daemon, but at this point all of the spee.ch code assumes the user has manually started the deamon already

Reply:
Did you have a different approach in mind?